menpofit/sdm/fitter.py

```diff
@@ -77,9 +77,8 @@ def _train(self, images, group=None, bounding_box_group=None,
 
         # In the case where group is None, we need to get the only key so that
         # we can attach landmarks below and not get a complaint about using None
-        first_image = first_batch[0]
```

Comment:
This seems to be a copy, so when `bb` are attached to the images in `image_batch` this `first_image` does not contain the landmark `group` used.